### ISSUE TITLE
BUG Submitting the Form block on the frontend fails

### ISSUE THREAD

**Issue by robbieaverill** [dnadesign/silverstripe-elemental-userforms#11]
I created a Form block with three text fields, published it then went to the frontend and tried to submit the form:

```
[Emergency] Uncaught BadMethodCallException: Object->__call(): the method 'Fields' does not exist on 'SilverStripe\UserForms\Control\UserDefinedFormController'
GET /home/element/20/Form?flushtoken=9abd47688233228f8c874c7542a47db7&flush=

Line 54 in /Users/robbieaverill/dev/ss4/vendor/silverstripe/framework/src/Core/CustomMethods.php
```

I've tested userforms 5.0.x-dev on a vanilla SS4 install without this module and it submits fine, so the problem must be in this module somewhere.

**Comment by wilr** [comment]
@robbieaverill - Have all my PR's merged for userforms. There was a required change on the trait to resolve that.

**Comment by robbieaverill** [comment]
@wilr as far as I can tell yes - there's the open PR for your update to the configurable template directory, but don't think that's related?

**Comment by robbieaverill** [comment]
Fix at #12
